css/src/components/messages.scss

```diff
@@ -0,0 +1,58 @@
+.messages {
+	display: flex;
+	flex-direction: column;
+	max-width: 100%;
+	gap: $layout-4;
+}
+
+.chat-bubble {
+	padding: $spacer-4;
+	border: $border-width solid $border !important;
+	border-radius: $border-radius !important;
+	box-shadow: $box-shadow-light !important;
+}
+
+.chat-bubble,
+.message-time {
+	width: 80%;
+}
+
+.message-time,
+.user-title {
+	color: $text-subtle;
```

Comment:
I could be wrong, but I'm guessing this won't meet contrast on primary.

Reply:
updated